xclim/indices/_conversion.py

```diff
@@ -928,6 +930,47 @@ def potential_evapotranspiration(
         out = (0.0023 * ra * (tas + 17.8) * (tasmax - tasmin) ** 0.5) / lv
         out = out.clip(0)
 
+    elif method in ["mcguinnessbordne05", "MB05"]:
+        if tas is None:
+            tasmin = convert_units_to(tasmin, "degC")
+            tasmax = convert_units_to(tasmax, "degC")
+            tas = (tasmin + tasmax) / 2
+            tas.attrs["units"] = "degC"
+
+        tas = convert_units_to(tas, "degC")
+        tasK = convert_units_to(tas, "K")
+
+        latr = (tas.lat * np.pi) / 180
+        jd_frac = (datetime_to_decimal_year(tas.time) % 1) * 2 * np.pi
+
+        S = 1367.0  # Set solar constant [W/m2]
+        ds = 0.409 * np.sin(jd_frac - 1.39)  # solar declination ds [radians]
+        omega = np.arccos(-np.tan(latr) * np.tan(ds))  # sunset hour angle [radians]
+        dr = 1.0 + 0.03344 * np.cos(
+            jd_frac - 0.048869
+        )  # Calculate relative distance to sun
+
+        ext_rad = (
+            S
+            * 86400
+            / np.pi
+            * dr
+            * (
+                omega * np.sin(ds) * np.sin(latr)
+                + np.sin(omega) * np.cos(ds) * np.cos(latr)
+            )
+        )
+        latentH = 4185.5 * (751.78 - 0.5655 * tasK)
+
+        radDIVlat = np.divide(ext_rad, latentH)
+
+        # parameters from calibration provided by Dr Maliko Tanguy @ CEH
+        # (calibrated for PET over the UK)
+        a = 0.00516409319477
+        b = 0.0874972822289
```

Comment:
How specific are these parameters? Is this method only adapted to the UK?
I suggest adding something in the docstring about this.

Reply:
I have added some notes and another reference to clarify.